### PR TITLE
Create Talk room for booked appointments

### PR DESCRIPTION
**Pull request by GretaD**
### ☑️ Resolves

* Fix https://github.com/nextcloud/documentation/issues/10022

### CODE REVIEW

**Comment by GretaD** [comment]
please run it locally if possible, because i dont see my new changes even tho the command` make SPHINXBUILD=sphinx-autobuild html` is run succesfully

File: user_manual/groupware/calendar.rst [hunk 1]
```diff
@@ -383,3 +383,8 @@ If the appointment has the setting "Add time before event" or "Add time after th
 As with any other event that has attendees, changes and cancellations will trigger a notification to the attendee's email.
 
 If attendees wish to cancel the appointment they have to get in contact with the organizer, so that the organizer can cancel or even delete the event.
+
+Create Talk room for booked appointments
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can create a Talk room directly from the calendar app for a booked appointment. The option can easily be found on the 'Create appointment' modal. A unique link will be generated for every booked appointment and sent via the confirmation email when you check this option.
```
Comment (ChristophWurst) [inline user_manual/groupware/calendar.rst:390]:
```suggestion
You can create a Talk room directly from the calendar app for a booked appointment. The option can be found on the 'Create appointment' modal. A unique link will be generated for every booked appointment and sent via the confirmation email when you check this option.
```

**Review by ChristophWurst — APPROVED** [review]
The unrelated whitespace diffs could cause backport conflicts

**Comment by GretaD** [comment]
> The unrelated whitespace diffs could cause backport conflicts

this is added by the make command, i will fix the conflicts on 26th by building the file again

**Pull request merged**